pkg/utils/constants.go

```diff
@@ -12,4 +14,14 @@ const (
 	MainnetChainId = 1
 	HoleskyChainId = 17000
 	LocalChainId   = 31337
+
+	MainnetNetworkName = "mainnet"
+	HoleskyNetworkName = "holesky"
+	PreprodNetworkName = "preprod"
```

Comment:
should also be `holesky`. Network would be either `preprod` or `testnet`

Reply:
updated 0aeb8f1a8894dac95c5306f67f45187b8d4b0e95